tests/testScripts/test_upcoming_events.py

```diff
@@ -0,0 +1,23 @@
+import config
+import pytest
+
+"""
+    Testing the upcoming public events
+"""
+
+def test_upcoming_public_events_guest(app):
```

Comment:
Try to use more informative method names like what it's supposed to test, what it should return.